gcloud/test_credentials.py

```diff
@@ -46,6 +46,7 @@ def _setup_appengine_import(test_case, app_identity):
     sys.modules['google.appengine.ext'] = GAE_EXT
     sys.modules['google.appengine.ext.webapp'] = GAE_EXT_WEBAPP
     sys.modules['google.appengine.ext.webapp.util'] = GAE_EXT_WEBAPP_UTIL
+    sys.modules['webapp2'] = GAE_EXT_WEBAPP
```

Comment:
You mean why was `google.appengine.ext.webapp` deprecated in favor of `webapp2`?

If that's what you meant, the support of that module as a web framework ceased in 2011 or 2010 and `webapp2` is the replacement.

If not, I'll try to address what you did mean.